sdk/storage/Azure.Storage.Blobs/src/BlobClientOptions.cs

```diff
@@ -69,8 +70,127 @@ public enum ServiceVersion
         /// </param>
         public BlobClientOptions(ServiceVersion version = LatestVersion)
         {
-            Version = version == ServiceVersion.V2019_02_02 ? version: throw Errors.VersionNotSupported(nameof(version));
+            Version = version == ServiceVersion.V2019_02_02 ? version : throw Errors.VersionNotSupported(nameof(version));
             this.Initialize();
+            List<string> LoggedHeaderNames = new List<string>
+            {
+                "Access-Control-Allow-Origin",
+                "x-ms-client-request-id",
```

Comment:
This is already in https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/DiagnosticsOptions.cs.  Can we make sure there aren't duplicates?

Reply:
Ahh..I just missed this one.